examples/boltzmann_wealth_model/app.py

```diff
@@ -0,0 +1,117 @@
+from boltzmann_wealth_model.model import BoltzmannWealthModel
+
+import streamlit as st
+
+import time
+
+import pandas as pd
+
+import altair as alt
+
+
+model = st.title("Boltzman Wealth Model")
+num_agents = st.slider(
+    "Choose how many agents to include in the model",
+    min_value=1,
+    max_value=100,
+    value=50,
+)
+num_ticks = st.slider(
+    "Select number of Simulation Runs", min_value=1, max_value=100, value=50
```

Comment:
Should be "Select number of simulation steps".

Reply:
the change is on the latest commit